### PR TITLE
[Player-3962] Not pausing videos on clicking subtitles/multi-language button

### PR DESCRIPTION
**Pull request by isheftel-ooyala**
[iOS][Android][OoyalaSkinSampleApp] Implemented not to pause any video on clicking subtitles/multi-language button, also when advertising starts if subtitles/multi-language menu is opened it will be closed.

iOS team, please first test on your devices, I was able to test on Android only.

### CODE REVIEW

File: sdk/react/ooyalaSkinCore.js [hunk 1]
```diff
@@ -121,7 +121,7 @@ OoyalaSkinCore.prototype.handlePress = function(n) {
       this.pushToOverlayStackAndMaybePause(OVERLAY_TYPES.DISCOVERY_SCREEN);
       break;
     case BUTTON_NAMES.AUDIO_AND_CC:
-      this.pushToOverlayStackAndMaybePause(OVERLAY_TYPES.AUDIO_AND_CC_SCREEN);
+      this.pushToOverlayStack(OVERLAY_TYPES.AUDIO_AND_CC_SCREEN);
       break;
```
Comment (SergeyBicarte) [inline sdk/react/ooyalaSkinCore.js:125]: Shouldn't this change applied to OoyalaSkinCore.prototype.handleMoreOptionsButtonPress function also?

Comment (isheftel-ooyala) [reply sdk/react/ooyalaSkinCore.js:125]: In ticket there was nothing about more options button.

Comment (SergeyBicarte) [reply sdk/react/ooyalaSkinCore.js:125]: That button (subtitles/multi-language) can be moved to MoreOption Panel dynamically. Will be it moved or not depends on current available size of skin layout and skin.json settings.

Comment (isheftel-ooyala) [reply sdk/react/ooyalaSkinCore.js]: done

**Review by SergeyBicarte — APPROVED** [review]
LGTM

**Review by MarkEPAM — APPROVED** [review]
LGTM

**Pull request merged**